docs/sftt/sftt-api-spec.md

```diff
@@ -1681,3 +1681,92 @@ If the site id specified is invalid.
 ##### `401 UNAUTHORIZED`
 
 If the calling user is not an admin.
+
+### Record Stewardship Activity
+
+`POST api/v1/protected/sites/:site_id/record_stewardship`
+
+Records a stewardship activity for the given site. Duration is the time taken to perform the activity in minutes. Indicate `True` if the activity (watered, mulched, etc.) was completed, else `False`.
+
+#### Request Body
+
+```json
+{
+  "date": DATE,
```

Comment:
Would we want to change this to `performed_on`/describe what date this is too?

Reply:
Probably yeah

Reply:
Opted for description